### ISSUE TITLE
Images transition

### ISSUE THREAD

**Issue by umbreak** [mpetroff/pannellum#46]
This is a style issue. It would be quite nice to have an animation effect when an image is loaded. Just the regular fadeIn (like Jquery) would be fair enough. There is an example [with canvas here](http://jsfiddle.net/m1erickson/zw9S4/). 

I was trying to implement it myself but I found out that this actually happens in libpannellum.js and is quite complex.

**Comment by umbreak** [comment]
Right now, after some fixes you made, the images transition does not work. Even setting the value to null. It appears a black image.

As far as I saw, removing the line 1343 in `pannellum.js` (where init() function is called)it solves the issue. 

But I think this init function is doing a lot of stuff which I guess is useful. I kept testing, and I see that after the renderer (line 209) is reasigned, the image shown turns black. It looks like is something messed up with that renderer instance (when changing the hotspots).

**Comment by mpetroff** [comment]
I am unable to reproduce this; the image transitions still work fine for me. Is it black just for the fade, or does it stay black? I'm guessing nothing is printed to the console. While I don't think it will make a difference in this case, set things to `undefined` instead of `null` as this is what the code compares against.

**Comment by umbreak** [comment]
If I don't put the option `sceneFadeDuration`, when I change the image, it shows a black image. The hotspots are there, and they're moving when I drag. But not the image.

**Comment by mpetroff** [comment]
I was able to track down the issue. With `autoLoad` specified in a tour, the `autoLoad` code was executed when each scene was loaded. This should never have happened, but it didn't affect anything before. A change I made caused this bug to manifest itself and break things; the extra call caused a second `canvas` to be created off-screen that displayed the image, while the on-screen `canvas` was blank. Now that I fixed it, things should work again.
